drivers/ninaw10/machine_pin_nina.c

```diff
@@ -98,4 +110,26 @@ void machine_pin_ext_config(machine_pin_obj_t *self, int mode, int value) {
     }
 }
 
+bool machine_pin_ext_is_adc(uint32_t channel) {
+    return NINA_GPIO_IS_ADC_CHANNEL(channel);
+}
+
+void machine_adc_ext_config(uint32_t channel) {
+    if (!NINA_GPIO_IS_ADC_CHANNEL(channel)) {
+        mp_raise_ValueError("Invalid ADC channel");
```

Comment:
I don't think this check is needed, this is an internal function and the channel arg should already be validated before passing it in here.